ksqldb-engine/src/main/java/io/confluent/ksql/ddl/commands/CreateSourceFactory.java

```diff
@@ -68,20 +72,32 @@ public CreateSourceFactory(final ServiceContext serviceContext) {
       final ServiceContext serviceContext,
       final SerdeOptionsSupplier serdeOptionsSupplier,
       final KeySerdeFactory keySerdeFactory,
-      final ValueSerdeFactory valueSerdeFactory
+      final ValueSerdeFactory valueSerdeFactory,
+      final MetaStore metaStore
   ) {
     this.serviceContext = Objects.requireNonNull(serviceContext, "serviceContext");
     this.serdeOptionsSupplier =
         Objects.requireNonNull(serdeOptionsSupplier, "serdeOptionsSupplier");
     this.keySerdeFactory = Objects.requireNonNull(keySerdeFactory, "keySerdeFactory");
     this.valueSerdeFactory = Objects.requireNonNull(valueSerdeFactory, "valueSerdeFactory");
+    this.metaStore = metaStore;
```

Comment:
nit: validate params that will be stored in object state; ensuring object does not get into an invalid state. i.e. use `requireNonNull`.

Reply:
this is done